src/components/portfolio/DetailsMainAnalyst.tsx

```diff
@@ -36,6 +36,13 @@ type DetailsMainAnalystProps = {
   positions: Position[];
 };
 
+/**
+ *
```

Comment:
Didn't we agree to not use @return and instead just write the purpose in normal doc string text? That's at least how I did it in my documentation for backtesting